module/documents/chat-message.mjs

```diff
@@ -120,6 +126,20 @@ function applyChatCardDamage(li, multiplier) {
   }));
 }
 
+/**
+ * Apply rolled dice as tempoary hit points to the controled token(s).
+ * @param {HTMLElement} li The chat entry which contains the roll data
+ * @returns {Promise}
+ */
+function applyChatCardTemp(li) {
+  const message = game.messages.get(li.data("messageId"));
+  const roll = message.roll;
```

Comment:
This throws a deprecation warning at the moment. Not sure what our overall strategy for these buttons when there are multiple rolls in a chat message will be, but for now you can just access the first roll directly:

```suggestion
  const roll = message.rolls[0];
```